job-service/src/main/java/com/hpe/caf/services/job/api/JobsGetById.java

```diff
@@ -65,8 +65,10 @@ public static Job getJob(final String partitionId, String jobId) throws Exceptio
             job = databaseHelper.getJob(partitionId, jobId);
 
             LOG.debug("getJobById: Done.");
-            LOG.info("Job progress {}/{} status {}  progression {}", partitionId, jobId, job.getStatus(), job.getPercentageComplete());
-            return job;
+            if (null != job) {
+                LOG.info("Job progress {}/{} status {}  progression {}", partitionId, jobId, job.getStatus(),
+                        job.getPercentageComplete());
+            }            return job;
```

Comment:
Merge `develop` into this branch to get rid of this change.

Reply:
Done